### DISCUSSION TITLE
3기 주간회의 - 멘토 제도 안내 및 멘토 희망자 신청

### DISCUSSION THREAD

**Discussion by gracefulBrown** [woowacourse/prolog#777] [Notice]
# 3기 크루들의 프롤로그 프로젝트 참여 방법
- 개발 참여 (기존 방법)
- 멘토로 참여 (새로운 방법)
> 개발을 하면서 동시에 멘토로 참여할 수 있습니다.

## 멘토의 역할
- 4기 크루들의 온보딩 시 멘토가 되어 질문이나 코드 설명 등 가이드를 해주어 적응을 돕습니다.
- (장기적으로) 코드 작업 시 코드 리뷰를 통해 도움을 줍니다.

## 멘토가 하는 일
- 4기 크루들은 파트 단위로 작업을 할당 받을 예정인데 이 파트에 멘토가 배정 됩니다.
- 멘토로 참여하시면 매칭된 파트에서 진행하는 이슈 및 업무를 도와주시면 됩니다.
  - 프로젝트 히스토리를 질문 할 수 있음
  - 코드를 분석할 때 도움을 요청할 수 있음
  - 코드 리뷰 요청 시 간단하게 리뷰를 할 수 있음

## 멘토 참여 방법
- 참여를 희망하실 경우 코멘트로 남겨주세요!
- **신청 마감은 이번주 22일 금요일 까지!!**

## 기타
- 이번이 멘토로 참여하시면 레벨2 기간 동안(6월중순까지) 멘토로 활동합니다.

**Comment by sihyung92** [comment]
멘토로 참가하겠습니다

**Comment by Hyeon9mak** [comment]
훈수 두겠습니다
